File: Serene/Serene.Script/Meeting/MeetingAgenda/MeetingAgendaDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingAgendaId"), NameProperty("Tags"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingAgenda"), LocalTextPrefix("Meeting.MeetingAgenda"), Service("Meeting/MeetingAgenda")]
    public class MeetingAgendaDialog : EntityDialog<MeetingAgendaRow>
    {
    }
}

File: Serene/Serene.Core/Modules/Meeting/MeetingAgenda/MeetingAgendaDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingAgendaDialog extends Serenity.EntityDialog<MeetingAgendaRow, any> {
        protected getFormKey() { return MeetingAgendaForm.formKey; }
        protected getIdProperty() { return MeetingAgendaRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAgendaRow.localTextPrefix; }
        protected getNameProperty() { return MeetingAgendaRow.nameProperty; }
        protected getService() { return MeetingAgendaService.baseUrl; }

        protected form = new MeetingAgendaForm(this.idPrefix);

    }
}